### ISSUE TITLE
Quests!

### ISSUE THREAD

**Issue by KVonGit** [KVonGit/HVACR-specifications#23]
### To add a Quest, enter this line:

```C++
AddQuest (DumbRH, "token", "Acquire a token from RH.")
```

```DumbRH``` is the NPC who has assigned the Quest. **(This MUST be the object's name. I.e., XanMag, DumbRH, DarkLizerd, or Dark_Lizerd.)**

```"token"``` will be the Quest's identifier.

```"Acquire a token from RH."``` is the message that will print in the list of active Quests.

---
### To close out a Quest, add this line the script where you would Increase the player's score:

```C++
QuestComplete (DumbRH, "token")
```

---
### You can enter QUEST to see a list of active Quests, OR you can click on the Active Quests bit in the status bar!

![image](https://user-images.githubusercontent.com/30656341/29894823-6faca148-8d9c-11e7-9055-93dd2b65baa5.png)

**Comment by XanMag** [comment]
Whoa.  Cool feature.

**Comment by KVonGit** [comment]
Thank ya', XM!

---
 I just committed version 0.0.30.

I'm checking it back in (at least for 5 minutes), and going to play with kvLib.

If kvLib is not included: the main game has an alternate RH and he'll grant you access to a very short run through his world. I also have it set so it won't throw any errors if kvLib is not included.

NOTE: If kvLib is not included, the title will have an asterisk displayed after it.

#### Pixie's Quest *


---
#### JUST FOR FUN:

If you want to look at some crazy coding, check out the ASK ABOUT script on DumbRH. (His world is created totally on the fly if you answer YES when he invites you to visit it. (Well... I can't figure out how to create a command on the fly, so there's a blank command in the beginning then I alter the pattern when the player examines the token for the first time. But, besides the command, everything is created on the fly upon entering RH' World. Only DumbRH and the command exist before you enter RH's World.))

**Comment by KVonGit** [comment]
Hot damn!

I wish I would have looked up something on Pixie's wiki before I wrote all this code!

Check it out:

https://github.com/ThePix/quest/wiki/Quest-Library

Damn it!

It appears to be what I basically did, but I forgot a FailedQuest function!!!

Working on the failed function now.

I'd rather not have a bunch of (or any) libraries...

The only reason I made my stuff into a library is because I kept messing you guys' fileses upses...
